src/main/java/org/opensearch/search/relevance/transformer/kendraintelligentranking/KendraIntelligentRanker.java

```diff
@@ -111,6 +111,10 @@ public SearchHits transform(final SearchHits hits,
         List<String> tokenizedTitle = null;
         if (queryParserResult.getTitleFieldName() != null) {
           tokenizedTitle = textTokenizer.tokenize(docSourceMap.get(queryParserResult.getTitleFieldName()).toString());
```

Comment:
I just checked and confirmed that the return value from `tokenize` is never null: https://github.com/opensearch-project/search-relevance/blob/main/src/main/java/org/opensearch/search/relevance/transformer/kendraintelligentranking/preprocess/TextTokenizer.java#L39